toptal/picasso
# feat(FileList): [TEA-2545] Implement internal file list component

[TEA-2545]

[DESIGN](https://share.goabstract.com/2556a122-6443-4546-96db-3d40c37b2a0d?mode=build&sha=0aeb05e01da837c228249c74cc79ec21280f8b6d)

### Description

Implement internal `FileList` component needed for `FileInput` component. Component will be added to `FileInput` as part of  [TEA-2546](https://toptal-core.atlassian.net/browse/TEA-2546).

> NOTE
>
> `FileList` example inside `FileInput` section is added just for testing purpose and will be removed as part of  [TEA-2546](https://toptal-core.atlassian.net/browse/TEA-2546)

### How to test

- In storybook, open `FileInput` component and find `FileList` section on the bottom
- Check if list of files is implemented according design

### Screenshots

<img width="347" alt="Screenshot 2021-05-12 at 02 14 06" src="https://user-images.githubusercontent.com/7594216/117899600-bcc20500-b2c7-11eb-938f-315b21fea1ae.png">

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>


[TEA-2545]: https://toptal-core.atlassian.net/browse/TEA-2545
